packages/jats-xml/src/utils.ts

```diff
@@ -25,9 +25,7 @@ export function convertToUnist(node: Element): GenericNode | GenericParent | und
       return {
         type: 'text',
         ...attributes,
-        value: String(text)
-          .replace(/\n(\s+)$/, '')
-          .replace('\n', ' '),
+        value: String(text).replace(/\n(\s+)$/, ''),
```

Comment:
Is this change significant?

Reply:
This was causing errors when reading ours vs pandocs export.

I think this needs to go to an option on how to treat spaces.